MSON Specification.md

```diff
@@ -794,6 +795,32 @@ by using an `optional` attribute and/or MAY indicate values are samples using a
     Implies an `array` type structure that MUST contain "red" as an item and MAY contain any other strings, where
     "green" is a sample value.
 
+With a `fixed-type` _[Type Attribute][]_:
+
+- If a _[Named Type][]_ or _[Member Type][]_ annotates its type as `fixed-type`, _[Nested Member Types][]_ do not inherit
+the `fixed-type` attribute.
+
+- An `array` based _[Named Type][]_ or _[Member Type][]_ MAY specify `fixed-type` to indicate the structure
+MUST contain items of the specified types only.
+
+    ```
+    - colors (array, fixed-type)
+        - (string)
+    ```
+
+    Implies a fixed-list `array` structure that MUST only contain string items.
+
+- An `object` based _[Named Type][]_ or _[Member Type][]_ MAY specify `fixed-type` to indicate an object where all
+the properties MUST be present. Further, such an `object` type structure MUST NOT contain any other properties.
+
+    ```
+    - person (object, fixed-type)
```

Comment:
I'd say both this and the example above are a bit unclear. The same can be achieved by using just `fixed`. As such it isn't clear why you would you `fixed-type` vs. `fixed`. I'd suggest to add a better example (probably with a value) to demonstrate the case for `fixed-type`